src/messages/MOSDOp.h

```diff
@@ -361,8 +361,7 @@ struct ceph_osd_request_head {
       encode(retry_attempt, payload);
       encode(features, payload);
     } else {
-      // latest v8 encoding with hobject_t hash separate from pgid, no
-      // reassert version
+      // latest v9 opentelemetry trace
```

Comment:
OK, here it is. Let's squash this fixup before merging.

Reply:
@rzarzynski which commit do you want me to squash?

Reply:
The first one: https://github.com/ceph/ceph/pull/47457/commits/a635a685c02f4b967243f15795fc305b305e71f4.

[Here](https://github.com/ceph/ceph/pull/47457/commits/a635a685c02f4b967243f15795fc305b305e71f4#r958387555) is the exchange that spurred the fixup, I think.